src/Integration.Vsix/SonarLintTagger/IIssueSpanCalculator.cs

```diff
@@ -26,8 +26,8 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     internal interface IIssueSpanCalculator
     {
         /// <summary>
-        /// Returns the text span corresponding to the supplied analysis issue
+        /// Returns the text span corresponding to the supplied analysis issue location
         /// </summary>
-        SnapshotSpan CalculateSpan(IAnalysisIssue issue, ITextSnapshot currentSnapshot);
+        SnapshotSpan CalculateSpan(IAnalysisIssueLocation location, ITextSnapshot currentSnapshot);
```

Comment:
I realised I was re-writing the span-to-translation calculation as part of the new buffer tagger, whereas this implementation already exists.
This refactoring makes the interface reusable in the buffer tagger.

Reply:
[Told you](https://github.com/SonarSource/sonarlint-visualstudio/pull/1610#discussion_r468640365) it should implement it 😛

Reply:
And I said `IAnalysisLocation` should have a `PrimaryLocation` property, but that involved more changes than I wanted to make at the moment...